contrib/completion/bash/docker

```diff
@@ -1465,6 +1514,246 @@ _docker_network() {
 	esac
 }
 
+_docker_service() {
+	local subcommands="
+		create
+		tasks
+		inspect
+		update
+		ls
+		rm
+	"
+	__docker_subcommands "$subcommands" && return
+
+	case "$cur" in
+		-*)
+			COMPREPLY=( $( compgen -W "--help" -- "$cur" ) )
+			;;
+		*)
+			COMPREPLY=( $( compgen -W "$subcommands" -- "$cur" ) )
+			;;
+	esac
+}
+
+_docker_service_create() {
+	case "$cur" in
+		-*)
+			COMPREPLY=( $( compgen -W "--help --constraint --endpoint-ingress --endpoint-mode --env --label --limit-cpu --limit-memory --mode --name --network --publish --reserve-cpu --reserve-memory --restart-condition --restart-delay --restart-max-attempts --restart-window --scale --stop-grace-period --update-delay --update-parallelism --user --volume --workdir" -- "$cur" ) )
+			;;
+	esac
+}
+
+_docker_service_ls() {
+	case "$cur" in
+		-*)
```

Comment:
ls doesn't have all those arguments. I thinks `update` and `ls` were inverted
`--scale` should be `--replicas`